test-suite-http-server-tck-function-aws-api-proxy-test/src/test/java/io/micronaut/http/server/tck/lambda/tests/MicronautLambdaHandlerHttpServerTestSuite.java

```diff
@@ -11,9 +11,8 @@
     "io.micronaut.http.server.tck.lambda.tests"
 })
 @ExcludeClassNamePatterns({
-        "io.micronaut.http.server.tck.tests.forms.FormUrlEncodedBodyInRequestFilterTest",
-        "io.micronaut.http.server.tck.tests.forms.FormsSubmissionsWithListsTest",
-        "io.micronaut.http.server.tck.tests.filter.options.OptionsFilterTest",
+    "io.micronaut.http.server.tck.tests.OctetTest",
```

Comment:
why does OctetTest of all things fail?

Reply:
it seems they payload fails base64 decoding: 

```
15:17:46.851 [HTTP-Dispatcher] ERROR i.m.f.a.p.test.AwsProxyHttpHandler - Illegal base64 character d
java.lang.IllegalArgumentException: Illegal base64 character d
```

https://github.com/micronaut-projects/micronaut-aws/blob/built-in-http-server/function-aws-api-proxy-test/src/main/java/io/micronaut/function/aws/proxy/test/AwsProxyHttpHandler.java#L70